.github/workflows/ci.yml

```diff
@@ -90,3 +90,12 @@ jobs:
           foreach ($package in $built_pkgs) {
               cpush -s "https://www.myget.org/F/vm-packages/api/v2" -k ${{ secrets.MYGET_TOKEN }} $package
           }
+      - name: Upload chocolatey logs to artifacts
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: logs-${{ matrix.os }}.zip
+          path: |
+            C:\ProgramData\chocolatey\logs\chocolatey.log
+            C:\ProgramData\chocolatey\lib-bad\**\tools\install_log.txt
+            C:\ProgramData\_VM\log.txt
```

Comment:
@mandiant/flare-vm any other useful log that we want to have access to?

Reply:
I believe these are all the relevant logs at the moment